user-interface/src/case-detail/panels/CaseDetailAuditHistory.test.tsx

```diff
@@ -0,0 +1,129 @@
+import { render, screen } from '@testing-library/react';
+import CaseDetailAuditHistory from '@/case-detail/panels/CaseDetailAuditHistory';
+import { CaseAuditHistory, CaseStaffAssignment } from '@/lib/type-declarations/chapter-15';
+
+const EXPECTED_DATE_TIME = '01/01/2024 12:01 PM MST';
```

Comment:
We are supposed to remove time from the display. Follow this through the UI and this test.